src/components/HubSpotForm/index.tsx

```diff
@@ -40,11 +40,17 @@ export class HubSpotForm extends React.Component<IHubSpotForm, IHubSpotFormState
     if (!(window as any).hbspt || !portalId || !formId) {
       return;
     }
+    const email = localStorage.getItem('email') || '';
```

Comment:
Suggest doing `window.localStorage` to be consistent with everywhere else

```suggestion
    const email = window.localStorage.getItem('email') || '';
```